allensdk/brain_observatory/behavior/behavior_ophys_metadata.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import pandas as pd
+from typing import Optional, Any
+
+from allensdk.brain_observatory.behavior.behavior_metadata import \
+    BehaviorMetadata
+
+
+class BehaviorOphysMetadata(BehaviorMetadata):
+    """Container class for behavior ophys metadata"""
+    def __init__(self, extractor: Any,
+                 stimulus_timestamps: np.ndarray,
+                 ophys_timestamps: np.ndarray,
+                 behavior_stimulus_file: pd.DataFrame):
+
+        """Note: cannot properly type extractor due to circular dependency
+        between extractor and transformer.
+        TODO fix circular dependency and update type"""
+
+        super().__init__(extractor=extractor,
+                         stimulus_timestamps=stimulus_timestamps,
+                         behavior_stimulus_file=behavior_stimulus_file)
+        self._extractor = extractor
+        self._ophys_timestamps = ophys_timestamps
+
+        # project_code needs to be excluded from comparison
+        # since it's only exposed internally
+        self._exclude_from_equals = {'project_code'}
+
+    @property
+    def emission_lambda(self) -> float:
+        return 520.0
+
+    @property
+    def excitation_lambda(self) -> float:
+        return 910.0
+
+    @property
+    def experiment_container_id(self) -> int:
+        return self._extractor.get_experiment_container_id()
+
+    @property
+    def field_of_view_height(self) -> int:
+        return self._extractor.get_field_of_view_shape()['height']
+
+    @property
+    def field_of_view_width(self) -> int:
+        return self._extractor.get_field_of_view_shape()['width']
+
+    @property
+    def imaging_depth(self) -> int:
+        return self._extractor.get_imaging_depth()
+
+    @property
+    def imaging_plane_group(self) -> Optional[int]:
+        return self._extractor.get_imaging_plane_group()
+
+    @property
+    def imaging_plane_group_count(self) -> int:
+        return self._extractor.get_plane_group_count()
+
+    @property
+    def indicator(self) -> str:
+        return 'GCAMP6f'
```

Comment:
I thought Marina did not want us to hard-code this in.

Reply:
@danielsf I plan on creating a new PR for this to merge into this PR, since this was a recent request.